### ISSUE TITLE
Move Foundational Accessibility Test Training video to Platform Website

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#32981]
## Issue Description
There are several new teams working on the VA.gov Platform. They are also new to the Collaboration Cycle. One of the requirements of the Collaboration Cycle is that teams complete their own Foundational Accessibility Testing prior to engaging in a Collaboration Cycle Staging Review. There is existing documentation on the Foundational testing (See [How to prepare for an accessibility staging review](https://depo-platform-documentation.scrollhelp.site/developer-docs/How-to-prepare-for-an-accessibility-staging-review.1909293137.html)) however the documentation alone is sometimes not sufficient and teams request more 1:1 support. 

In February of 2021, former Platform Accessibility Specialist Trevor Pierce recorded a training of how to execute the Foundational Accessibility Tests. The [recording of the training](https://zoom.us/rec/play/JKroigFVBqkqWMgEr2Bn0wWaMGlD7RraNi4le83sTG-aqoAH4ad0weyBEOqb_rIyEOLuE6yN8z7EXPE-.es6Ywn1aJIqT9kE4?autoplay=true&startTime=1613667844000) is still available. It would be valuable to add this video as a reference for VFS teams to the Platform Website page referenced above.

---
## Tasks
- [x] Segment the video into 6 parts: Intro, 5 types of tests
- [x] Reference this page: https://depo-platform-documentation.scrollhelp.site/developer-docs/WCAG-2.1-success-criteria-and-foundational-testing.1777893393.html
- [x] Contact Content & IA team about how to get access to Platform YouTube Channel
- [x] Search Confluence for YouTube info.

## Acceptance Criteria
- [x] The video is available on Platform Website for VFS team use.
- [x] VFS teams have been made aware of this video through appropriate communication channels.

---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `tools-be`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by seanasewell** [comment]
@shiragoodman I segmented the video into multiple parts. Is there someone specific on the Content & IA team I should reach out to in order to get access to the Platform Youtube channel? And if not, what channel should I post in on Slack?

**Comment by shiragoodman** [comment]
@seanasewell I found this guidance on Platform website: https://vfs.atlassian.net/wiki/spaces/LEAD/pages/1847820659/Platform+YouTube+channel

It says all leads have access to add people.. so this would be Andrea, Michael Fleet and Naomi (I checked to see if I had the ability to add people and I do not). I would start with Naomi.

**Comment by seanasewell** [comment]
@noahgelman I have edited and added all the foundational accessibility test videos to the platform Youtube channel. I added all the links to the "How to prepare for an accessibility staging review" confluence document. You can access the specific location [here](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1909293137/How+to+prepare+for+an+accessibility+staging+review#Foundational-accessibility-tests-(required)). Can you take a quick look at the Confluence doc? I just wanted to make sure I put them in the right place.

@shiragoodman Once he gets his eyes on it, is it safe to publish to the VA Platform website?

**Comment by shiragoodman** [comment]
@seanasewell sounds good. Can you please clarify where in the doc you made changes so it's easy for us to see? Possibly just add a comment to each change?

Upon first glance, I noticed that some of the content on this page may be outdated (for example DEPO opposed to OCTO, references to Nightwatch, which I think has been retired). I am going to ping the Testing Tools Team to see if they can take a look at this before we publish.

**Comment by shiragoodman** [comment]
@seanasewell also, did you make a video exert of the introduction? It might be nice to have the whole video series available to view.

**Comment by seanasewell** [comment]
@shiragoodman https://vfs.atlassian.net/wiki/spaces/pilot/pages/1909293137/How+to+prepare+for+an+accessibility+staging+review#Foundational-accessibility-tests-(required)

I just added the videos to the top under the title "Foundational accessibility tests" (see link above). I also made a comment in confluence where the videos were added in the document. Let me know your thoughts to the placement of the videos.

**Comment by shiragoodman** [comment]
@seanasewell i think the video placement is good enough for now. In the page I asked you to reference under Tasks, the videos are embedded into the page, not linked out. however, I think the way you have it organized is sufficient.

I did ask in the Confluence doc that you update some of this links (as they're incorrectly linking to github). Let me know when that's updated please.

**Comment by seanasewell** [comment]
@shiragoodman **Status Update** I updated "Read More" links to be anchor links and I replaced the text "DEPO" with "OCTO-DE".

**Comment by shiragoodman** [comment]
thanks @seanasewell  - you can publish to Platform Website. I also added a post to the Change History so when you publish, you'll see a change was made there too. let me know if you have any question.

**Comment by seanasewell** [comment]
@shiragoodman Part of the success criteria for this issue is that "VFS teams have been made aware of this video through appropriate communication channels". Should I post in a specific channel that these videos have been added to the Platform Youtube channel?

**Comment by shiragoodman** [comment]
@seanasewell good questions. Per my comment above, I added a post to our[ Change History](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Change-history.1846870032.html) page. When you published the updates to Platform Website yesterday, you also published this page. Allison we'll be wrapping up Calendly work soon. When she does and we go live, I'll announce both changes together.

For future reference, any time "appropriate communication channels" are mentioned wrt Collab Cycle, it usually means 3 things:
1. A post has been added to the Change History log
2. We've shared on #vfs-all-teams
3. I've shared at the weekly Team of Teams meeting

**Comment by shiragoodman** [comment]
@seanasewell you can close this
